### PR TITLE
[bulk] validation: validation edits JM 5/12a

### PR DESCRIPTION
**Pull request by jmart1428**
This PR fixes 60 issues.

### CODE REVIEW

File: aspnet/mvc/overview/getting-started/introduction/adding-search.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/getting-started/introduction/adding-search
 title: "Search | Microsoft Docs"
 author: Rick-Anderson
-description: ""
+description: "In this section you'll add search capability to the `Index` action method that lets you search movies by genre or name."
```
Comment (LSchaufMS) [inline aspnet/mvc/overview/getting-started/introduction/adding-search.md:5]: Change " `Index` " to " Index " (remove backticks from the description).

File: aspnet/mvc/overview/getting-started/introduction/adding-validation.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/getting-started/introduction/adding-validation
 title: "Adding Validation | Microsoft Docs"
 author: Rick-Anderson
-description: ""
+description: "In this section you'll add validation logic to the `Movie` model, and you'll ensure that the validation rules are enforced any time a user attempts to create or edit a movie using the application."
```
Comment (LSchaufMS) [inline aspnet/mvc/overview/getting-started/introduction/adding-validation.md:5]: Remove backticks from the description: " `Movie` " to " Movie "

File: aspnet/mvc/overview/getting-started/introduction/examining-the-details-and-delete-methods.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
 ---
 uid: mvc/overview/getting-started/introduction/examining-the-details-and-delete-methods
-title: "Examining the Details and Delete Methods | Microsoft Docs"
+title: "Examining the MVC Details and Delete Methods | Microsoft Docs"
 author: Rick-Anderson
-description: ""
+description: "In this tutorial, you'll examine the automatically generated `Details` and `Delete` methods."
```
Comment (LSchaufMS) [inline aspnet/mvc/overview/getting-started/introduction/examining-the-details-and-delete-methods.md:5]: Remove backticks from the description.

File: aspnet/mvc/overview/getting-started/introduction/examining-the-edit-methods-and-edit-view.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/getting-started/introduction/examining-the-edit-methods-and-edit-view
 title: "Examining the Edit Action Methods and Views for the Movie Controller | Microsoft Docs"
 author: Rick-Anderson
-description: ""
+description: "In this tutorial, you'll examine the generated `Edit` action methods and views for the movie controller."
```
Comment (LSchaufMS) [inline aspnet/mvc/overview/getting-started/introduction/examining-the-edit-methods-and-edit-view.md:5]: Remove backticks from the description.

**Comment by jmart1428** [comment]
**Global effort to fix build validation errors**
@Rick-Anderson 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**